packages/storybook-vue/src/stories/Heading/Heading.stories.mdx

```diff
@@ -10,93 +10,46 @@ export const Template = (args) => ({
   setup() {
     return { args };
   },
-  template: `<Heading v-bind="args">${args.default}</Heading>`,
+  template: `
+  <Heading v-bind="args" :level="args.level1">${args.headingContentLevel1}</Heading>
+  <Heading v-bind="args" :level="args.level2">${args.headingContentLevel2}</Heading>
+  <Heading v-bind="args" :level="args.level3">${args.headingContentLevel3}</Heading>
+  <Heading v-bind="args" :level="args.level4">${args.headingContentLevel4}</Heading>
+  <Heading v-bind="args" :level="args.level5">${args.headingContentLevel5}</Heading>
+  <Heading v-bind="args" :level="args.level6">${args.headingContentLevel6}</Heading>
+  `,
 });
 
 <Canvas>
   <Story
-    name="Heading 1"
+    name="Heading level 1-6"
     args={{
-      level: 1,
-      default: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      level1: 1,
+      level2: 2,
+      level3: 3,
+      level4: 4,
+      level5: 5,
+      level6: 6,
+      headingContentLevel1: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      headingContentLevel2: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      headingContentLevel3: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      headingContentLevel4: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      headingContentLevel5: "The Quick Brown Fox Jumps Over The Lazy Dog",
+      headingContentLevel6: "The Quick Brown Fox Jumps Over The Lazy Dog",
```

Comment:
How about using only `textContent` property that is used for every header? You probably want to compare the same text in different fonts anyway.

Reply:
Honestly, I don't think we need a property here, hardcoded text is more than enough  because the story role is just to display all heading in one page, like I said in the previous comment we have the controls individually